mlflow/pipelines/steps/evaluate.py

```diff
@@ -60,6 +60,12 @@ def _validate_and_apply_step_config(self):
                 "Missing template_name config in pipeline config.",
                 error_code=INVALID_PARAMETER_VALUE,
             )
+        self.positive_class = self.step_config.get("positive_class")
```

Comment:
Shall we verify that positive_class actually exists (i.e. a valid label) in the dataset[target_col] in the ingest step?

Reply:
Good question, I think it depends. As things stand, the `positive_class` does not need to exist in the raw data, it could be created during the transform step. Do we want to require that `positive_class` exists in the raw data? I don't immediately see a reason to do so, but please challenge me on this!

We could also alternatively verify that `positive_class` exists after the transform step.

Let me know what you think!

Reply:
Let's don't add anything yet given we don't have any user feedback. We can always revisit this decision later on.

Reply:
As I understand it, the transform step is only used for feature transformations. Transformations to target columns should be performed upstream (e.g. in ETL). The regression pipeline doesn't pass targets to the transform step.

Reply:
`The regression pipeline doesn't pass targets to the transform step, for example.`
-> Right you are, my apologies. Would you like to see this validation performed during ingest (or split?)

Reply:
No worries! I think ingest is a good place for this.